awsconfigs/common/aws-authservice/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+)
+
+var port string
+var redirectURL string
+
+func init() {
+	port = os.Getenv("PORT")
+	redirectURL = os.Getenv("LOGOUT_URL")
+
+}
+
+// LogoutHandler expires ALB Cookies and redirects to Cognito Logout Endpoint
+func LogoutHandler(w http.ResponseWriter, r *http.Request) {
+	log.Println("Traffic reached LogoutHandler")
+
+	// There are 4 possible AWSELBAuthSessionCookies
+	for cookieIndex := 0; cookieIndex < 4; cookieIndex++ {
```

Comment:
is there a way to dynamically retrieve the number of cookies ? I am afraid of this magic number `4`.

Reply:
I looked into this and it seems like the `.Cookies()` method  that should retrieve the cookies that exist appears to be empty so, I'm leaning towards it may not be possible.